app/factory/Layer.js

```diff
@@ -325,7 +332,11 @@ Ext.define('CpsiMapview.factory.Layer', {
             isTimeDependent: !!layerConf.timeitem,
             dateFormat: layerConf.dateFormat,
             timeProperty: layerConf.timeitem,
-            isWfs: true
+            isWfs: true,
+            // TODO docs
+            // TODO wouldn't it make sense to have the actual field here
+            //      instead of at the slider and globally for all layers?
```

Comment:
Totally agreed. It would be better to have the field in here and detect `isNumericDependent` based on the existence of this field.

Reply:
This something that @geographika has to say something to, I think. This can be changed afterwards in any way we want it.

Reply:
Agree far more flexible to have the field set at the layer level rather than the component.

Reply:
I'd like to do this in a follow-up PR if that is fine for you.

Reply:
Fine for me. Maybe we open an issue for that, so we keep it in mind?

Reply:
Please see here for my thoughts on the topic: https://github.com/meggsimum/cpsi-mapview/issues/99